src/main/kotlin/blackjack/controller/BlackJack.kt

```diff
@@ -0,0 +1,40 @@
+package blackjack.controller
+
+import blackjack.model.Dealer
+import blackjack.model.Player
+import blackjack.view.View
+
+class BlackJack {
+
+    fun run() {
+        val players = View.inputPlayerNames()
+        val dealer = Dealer()
+
+        players.forEach {
+            it.addCards(dealer.dealingTwoCards())
+        }
+        View.initialCardDealingComment(players)
+        players.forEach { View.showCards(it) }
+
+        while (!players.noMoreHit()) {
```

Comment:
```suggestion
        while (players.noMoreHit().not()) {
```
자바에서는 !만을 제공하지만 코틀린에서는 `not()`을 제공하기 때문에 조금 더 읽히기 쉬운 코드를 작성할 수 있습니다.